gradle.properties

```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.3.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 9
```

Comment:
We should not need to change the API revision for this change. It is not a breaking change.

Reply:
Thanks @josephzunigadaly. I was playing it safe to avoid disrupting other people's work.

Reply:
Changing this disrupts people that do need to change `cordaApiRevision`. Please don't change this unless it is a breaking change.

Reply:
I think I would agree with @josephzunigadaly on this instance - let's not bump-up `cordaApiRevision` when we do not need to.

It will also make our life easier when integrating this change, as we will not have to apply for `corda-api` lock.